noir_stdlib/src/meta/op.nr

```diff
@@ -39,7 +39,8 @@ impl UnaryOp {
         } else if self.is_dereference() {
             quote { * }
         } else {
-            crate::mem::zeroed()
+            let op = self;
+            std::panic::panic(f"Unexpected unary operator in UnaryOp::quoted: {op}")
```

Comment:
The errors you're getting are because we have to use `crate::` for the current crate rather than std
```suggestion
            crate::panic::panic(f"Unexpected unary operator in UnaryOp::quoted: {op}")
```